src/data-workspace/display/table-custom-data-set.js

```diff
@@ -0,0 +1,104 @@
+import {
+    DataTable,
+    TableHead,
+    DataTableRow,
+    DataTableColumnHeader,
+    TableBody,
+    DataTableCell,
+} from '@dhis2/ui'
+import DOMPurify from 'dompurify'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './table.module.css'
+
+// Needs to have the same width as the table, so can't use the one from
+// @dhis2/ui
+const DataTableToolbar = ({ children, columns }) => (
```

Comment:
Minor: It's not immediately obvious that `columns` is a number here. `columnCount` would be more descriptive maybe?